lib/src/functions/math.dart

```diff
@@ -21,14 +22,16 @@ final global = UnmodifiableListView([
   _round, _ceil, _floor, _abs, _max, _min, _randomFunction, _unit, //
   _percentage,
   _isUnitless.withName("unitless"),
-  _compatible.withName("comparable")
+  _compatible.withName("comparable"),
+  _div.withName("divide")
```

Comment:
As with the spec CL, I'd leave the addition of the division function to later.

Reply:
Done.